src/app/layout.tsx

```diff
@@ -0,0 +1,43 @@
+import localFont from "@next/font/local";
+import "../../styles/globals.css";
+
+import { createServer } from "miragejs";
+import { ReactNode } from "react";
+
+export const blenderBook = localFont({
+  src: "./fonts/BlenderPro-Book.woff2",
+  variable: "--blender-book",
+});
+export const blenderBold = localFont({
+  src: "./fonts/BlenderPro-Bold.woff2",
+  variable: "--blender-bold",
+});
+
+export default function Layout({ children }: { children: ReactNode }) {
+  createServer({
+    routes() {
+      this.get("/api/footer/links", () => [
+        "thewitcher.com",
+        "playgwent.com",
+        "eu.gear.cdprojektred.com",
+      ]);
+
+      this.passthrough("/_next/static/development/_devMiddlewareManifest.json");
+    },
+  });
```

Comment:
This should be abstracted to a separate file related just to mirage, what do you think?